app/src/main/java/com/eventyay/organizer/core/faq/create/CreateFaqFragment.java

```diff
@@ -44,6 +44,9 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         validator = new Validator(binding.form);
 
         binding.submit.setOnClickListener(view -> {
+            binding.form.faqQuestion.setText(binding.form.faqQuestion.getText().toString().trim());
+            binding.form.faqAnswer.setText(binding.form.faqAnswer.getText().toString().trim());
```

Comment:
Just trim in createFaq method

Reply:
I think that it can't be done in the ViewModel since validation is being done through the fragment.